python/feathub/metric_stores/metric_store.py

```diff
@@ -47,6 +51,14 @@ def __init__(self, config: Dict) -> None:
         """
         self._metric_store_config = MetricStoreConfig(config)
 
+        # TODO: Add validators to verify the precision of configurations
+        #  and validators to express and/or logic.
+        interval = self._metric_store_config.get(
+            METRIC_STORE_REPORT_INTERVAL_SEC_CONFIG
+        )
+        if not interval.is_integer():
```

Comment:
Do we still need this check?

Reply:
Given that Feathub is supposed to support intervals whose precision is at most in seconds, this code helps to identify invalid configurations like `report_interval_sec = 5.5`.

Reply:
Based on the offline discussion, we have agreed to remove this check.